README.md

```diff
@@ -1,11 +1,11 @@
 
-ECMAScript2015(ES6) is the latest version of the ECMAScript standard. 
+ECMAScript 2016 (ES7) is the latest version of the ECMAScript standard. 
```

Comment:
I think we are emphasis on ES6 features here. Could you think about a better way for this?